parachains/runtimes/assets/asset-hub-westend/src/xcm_config.rs

```diff
@@ -593,3 +600,80 @@ where
 		sp_std::boxed::Box::new(Self::asset_id(asset_id))
 	}
 }
+
+/// All configuration related to bridging
+pub mod bridging {
+	use super::*;
+	use assets_common::{matching, matching::*};
+	use sp_std::collections::btree_set::BTreeSet;
+	use xcm_builder::{NetworkExportTable, UnpaidRemoteExporter};
+
+	parameter_types! {
+		// used for local testing
+		pub BridgeHubParaId: u32 = 1002;
+		pub BridgeHub: MultiLocation = MultiLocation::new(1, X1(Parachain(BridgeHubParaId::get())));
+		pub const KusamaLocalNetwork: NetworkId = NetworkId::Kusama;
```

Comment:
Why is this Kusama and not Westend?

Reply:
because, we dont have dedicated runtime for AssetHub for Rococo, and we use there AssetHubKusama runtime in Rococo, for Wococo we use AssetHubWestend runtime - yes, little bit of chaos :)

but anyway, I reverted all AssetHubWestend changes here, they are not really needed (and Rococo/Wococo we will fix other way)